protected/humhub/modules/stream/models/StreamQuery.php

```diff
@@ -140,7 +141,8 @@ class StreamQuery extends Model
         BlockedUsersStreamFilter::class,
         DateStreamFilter::class,
         DraftContentStreamFilter::class,
-        ScheduledContentStreamFilter::class
+        ScheduledContentStreamFilter::class,
+        ModuleStreamFilter::class,
```

Comment:
@funkycram I think init of this filter should be moved into `humhub\modules\stream\models\ContentContainerStreamQuery` because only this query class has the property `$container` which is required for the filter.
We need it at least for fixing tests here https://github.com/humhub/humhub/actions/runs/5866247719/job/15904674985#step:18:1437:

![tests_stream_query](https://github.com/humhub/humhub/assets/6995524/c0bbf823-3309-4e68-acdd-b3057046d94d)

The `ContentContainerStreamQuery` can be updated with this code:

```php
/**
 * @inheritdoc
 */
public function init()
{
    $this->filterHandlers[] = ModuleStreamFilter::class;
    parent::init();
}
```

Reply:
Yes, good point @yurabakhtin done in commit https://github.com/humhub/humhub/pull/6499/commits/1813ae3b8fbf8e568aa6551a7e235988e729dd8a

Reply:
@yurabakhtin we still have the same error from `StreamQueryTest`:
Getting unknown property: humhub\modules\stream\models\StreamQuery::container  

I don't understand what's wrong as there are no change in `StreamQuery`. Any idea?

Reply:
@funkycram I see after moved the filter here https://github.com/humhub/humhub/pull/6499/commits/1813ae3b8fbf8e568aa6551a7e235988e729dd8a the error is gone.

Reply:
Fine. So it's just that I hadn't waited long enough!